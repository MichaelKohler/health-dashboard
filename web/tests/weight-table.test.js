/* global it, expect */
/* eslint-disable react/jsx-filename-extension */

import React from 'react';
import { shallowToJson  } from 'enzyme-to-json';
import { shallow } from 'enzyme';
import WeightTable from '../src/weight-table.jsx';

jest.mock('react-redux', () => ({
  useDispatch: () => jest.fn(),
}));

it('renders correctly', () => {
  const data = [{
    createdAt: '2019-08-07T08:00:03Z',
    weight: 80,
  }];

  const wrapper = shallow(<WeightTable rows={ data }/>);
  expect(shallowToJson(wrapper)).toMatchSnapshot();
});

/* eslint-enable react/jsx-filename-extension */
