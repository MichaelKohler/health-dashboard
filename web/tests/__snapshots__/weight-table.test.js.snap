// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="MuiPaper-root MuiPaper-elevation1 makeStyles-root-1 MuiPaper-rounded"
>
  <table
    className="MuiTable-root makeStyles-table-2"
  >
    <thead
      className="MuiTableHead-root"
    >
      <tr
        className="MuiTableRow-root MuiTableRow-head"
      >
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-alignLeft MuiTableCell-sizeSmall MuiTableCell-head"
          scope="col"
        >
          Date
        </th>
        <th
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-alignRight MuiTableCell-sizeSmall MuiTableCell-head"
          scope="col"
        >
          Weight
        </th>
      </tr>
    </thead>
    <tbody
      className="MuiTableBody-root"
    >
      <tr
        className="MuiTableRow-root"
      >
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-alignLeft MuiTableCell-sizeSmall MuiTableCell-body"
        >
          Wed Aug 07 2019 10:00:03 GMT+0200 (Central European Summer Time)
        </td>
        <td
          aria-sort={null}
          className="MuiTableCell-root MuiTableCell-alignRight MuiTableCell-sizeSmall MuiTableCell-body"
        >
          80
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
